src/subdomains/generic/kyc/services/kyc.service.ts

```diff
@@ -266,16 +266,24 @@ export class KycService {
     return KycStepMapper.toKycResult(kycStep);
   }
 
-  async updateUserData(
+  async updateKycStep(
     kycHash: string,
     stepId: number,
     data: Partial<UserData>,
     requiresInternalReview: boolean,
+    updateUserData: boolean,
```

Comment:
@davidleomay makes one more param sense or should we just make two methods?

Reply:
I think 2 methods would be cleaner 🤔

Reply:
Yeah thought that too, it's a bit strange to build it in 2 methods because it's only 1 line that differs (the updateUserData line) and above and below the code is the same in both methods 😅

Reply:
I have pushed another solution you can look at it maybe it's acceptable too 👍